core/src/engine.cpp

```diff
@@ -20,6 +22,9 @@ void Engine::update_modules() {
     for (auto& module : m_engineModules) {
         module->on_update(m_windowModule->get_delta_time());
     }
+    for (auto& module : m_engineModules) {
```

Comment:
Should this only be called on a counter? count += deltaTime;
everytime a countMax is reached call fixed_update
Could also make this adaptable if the game is struggling?

Reply:
the fixed update is a frame-rate independent message for physics calculations.